Client/Card/index.ts

```diff
@@ -2,26 +2,29 @@ import * as model from "../../model"
 import { Connection } from "../Connection"
 import { Resource } from "../Resource"
 
-export class Card extends Resource<model.CardResponseV2, model.CreateCardRequest> {
-	constructor(connection: Connection, folder: string, backend: model.CardResponseV2) {
+export class Card extends Resource<model.CardResponseV2 | model.CardResponse, model.CreateCardRequest> {
+	constructor(connection: Connection, folder: string, backend: model.CardResponseV2 | model.CardResponse) {
 		super(connection, folder, backend)
 	}
 	amend(request: model.AmendCardRequest) {
-		return this.connection.post<model.CardResponseV2>(`${this.folder}/amend`, request)
+		return this.connection.post<model.CardResponseV2 | model.CardResponse>(`${this.folder}/amend`, request)
```

Comment:
these classes were all made incorrectly, none of the endpoints other than /v2/cards/virtual makes a v2 response